packages/middleware-eventstream/src/pluggin.ts

```diff
@@ -0,0 +1,25 @@
+import { Pluggable } from "@aws-sdk/types";
```

Comment:
rename file to `plugin.ts`?

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/1082/commits/8e9bbc011160d4bf0fd0e799cf4873a875d95c12